commonlibktx/src/androidTest/java/com/crisolutions/commonlibraryktx/RxTests.kt

```diff
@@ -33,51 +31,39 @@ class RxTests {
     fun testRelayOnTrue() {
 
         val testRelay = BehaviorRelay.createDefault(true)
-        var testVariable = 0
 
         testRelay.relayOnTrue {
-            testVariable = 1
+            pass()
         }
-
-        assertTrue(testVariable == 1)
     }
 
     @Test
     fun testRelayOnTrueFailure() {
 
         val testRelay = BehaviorRelay.createDefault(false)
-        var testVariable = 0
 
         testRelay.relayOnTrue {
-            testVariable = 1
+            fail()
         }
-
-        assertFalse(testVariable == 1)
     }
 
     @Test
     fun testRelayOnFalse() {
 
         val testRelay = BehaviorRelay.createDefault(false)
-        var testVariable = 0
 
         testRelay.relayOnFalse {
-            testVariable = 1
+            pass()
```

Comment:
If you comment this line out, does the test still pass? I don't know if it's doing anything. You might want to keep what you had (and maybe refactor it to use a boolean instead of an int).

Reply:
Yea, updated using booleans instead, that's a good idea.